### ISSUE TITLE
Display tags/labels of blog posts

### ISSUE THREAD

**Issue by lunkwill42** [Uninett/nav-landing-page#35]
#34 cleans up the tags of the blog posts somewhat. However, there isn't currently any template code to display post tags.

Tags do not need to be prominent in a blog post (they probably belong in an `<aside>` at the end of each blog post body). An HTML list of tags would do, and styled as grey-ish "labels" beneath each blog post (i.e. no need for fancy colors).

There is also currently no way to navigate posts by tag, but this is a separate issue.

**Comment by lunkwill42** [comment]
Something like this after `{{ .Content }}` in `full-blog-body.html` might do:

```gotemplte
        {{ if .Params.Tags }}
        <aside class="tags">
          <ul>
            {{ range .Params.Tags }}
            <li>{{ . }}</li>
            {{ end }}
          </ul>  
        </aside>
        
        {{ end }}

```

**Comment by podliashanyk** [comment]
> There is also currently no way to navigate posts by tag, but this is a separate issue.

Created #38

**Comment by lunkwill42** [comment]
Fixed by #41
